api/client/proto/authservice.proto

```diff
@@ -1368,9 +1372,14 @@ message PaginatedResource {
     // Resource is the resource itself.
     oneof resource {
         // DatabaseServer represents a DatabaseServer resource.
-        types.DatabaseServerV3 DatabaseServer = 1;
-        // ApplicationServer represents a AppServer resource.
-        types.AppServerV3 AppServer = 2;
+        types.DatabaseServerV3 DatabaseServer = 1
+            [ (gogoproto.jsontag) = "database_server,omitempty" ];
+        // AppServer represents a AppServer resource.
+        types.AppServerV3 AppServer = 2 [ (gogoproto.jsontag) = "app_server,omitempty" ];
```

Comment:
I'm not sure we can set jsontags here now since it wouldn't be a backwards compatible change.

Reply:
True, I will keep the fields only present at the new ones.